cartesi-rollups/api/backend/introduction.md

```diff
@@ -135,7 +136,7 @@ In the dApp architecture, here is how an advance request plays out.
 
 - Step 1: Send an input to the [`addInput(address, bytes)`](../json-rpc/input-box.md/#addinput) function of the InputBox smart contract.
 
-- Step 2: The HTTP Rollups Server reads the data and gives it for processing in the Cartesi machine.
+- Step 2: The HTTP Rollups Server reads the data and gives it to the Cartesi machine for processing.
 
 - Step 3: After the computation, the state is updated, and the results are returned to the rollup server. 
```

Comment:
"the state is updated" to "the machine state is updated"

Reply:
Why is this resolved?